packages/vats/src/bridge-target.js

```diff
@@ -0,0 +1,153 @@
+// @ts-check
+import { E } from '@endo/far';
+import { M } from '@endo/patterns';
+
+import { BridgeHandlerI } from './bridge.js';
+
+const { Fail } = assert;
+
+/**
+ * @typedef {object} App
+ * @property {(obj: any) => Promise<unknown>} upcall return value depends on the
+ *   bridge semantics
+ */
+export const AppI = M.interface('App', {
```

Comment:
without any gap, the typedef block looks like an annotation on this but it isn't. maybe we need a utility type so we could have an annotation like,
```
/** @type {GuardedInterface<App>} */
```

I've made a ticket: https://github.com/Agoric/agoric-sdk/issues/9163

meanwhile, please add a comment or new line so the typedef doesn't appear to be an annotation. e.g.,
```
// TODO unwrap type https://github.com/Agoric/agoric-sdk/issues/9163
```